tiatoolbox/dataloader/slide_info.py

```diff
@@ -0,0 +1,48 @@
+"""Get Slide Meta Data information"""
+from tiatoolbox.dataloader import wsireader
+from tiatoolbox.decorators.multiproc import TIAMultiProcess
+
+import os
+
+
+@TIAMultiProcess(iter_on="input_path")
+def slide_info(input_path, output_dir=None):
+    """Single file run to output or save WSI meta data. Multiprocessing uses this function
```

Comment:
Should be a line break after the three opening quotes here

Reply:
No line break in Google Style
https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings
https://github.com/TIA-Lab/tiatoolbox/commit/8af253ae818e1178b40e511fc63843fbd5c4e82a